### ISSUE TITLE
Deleting a language

### ISSUE THREAD

**Issue by alaplap** [opencart/opencart#4768]
Hi!

In v2.3.0.2 deleting a language causes a fatal error:

Fatal error: Uncaught exception 'Exception' with message 'Error: Table 'oc_teszt.oc_banner_image_description' doesn't exist<br />Error No: 1146<br />DELETE FROM oc_banner_image_description WHERE language_id = '1'' in C:\xampp\htdocs\opencart\system\library\db\mysqli.php:40 Stack trace: #0 C:\xampp\htdocs\opencart\system\library\db.php(16): DB\MySQLi->query('DELETE FROM oc_...', Array) #1 C:\xampp\htdocs\opencart\admin\model\localisation\language.php(222): DB->query('DELETE FROM oc_...') #2 [internal function]: ModelLocalisationLanguage->deleteLanguage('1') #3 C:\xampp\htdocs\opencart\system\engine\loader.php(178): call_user_func_array(Array, Array) #4 [internal function]: Loader->{closure}(Array, Array) #5 C:\xampp\htdocs\opencart\system\engine\proxy.php(25): call_user_func_array(Object(Closure), Array) #6 C:\xampp\htdocs\opencart\admin\controller\localisation\language.php(88): Proxy->__call('deleteLanguage', Array) #7 C:\xampp\htdocs\opencart\admin\controller\localisation\language.php(88): Proxy->deleteLanguage('1') #8 [ in C:\xampp\htdocs\opencart\system\library\db\mysqli.php on line 40

**Comment by andrei-bogdan** [comment]
Hi, 

Please send more information about this issue, in order for us to try to replicate it (including screenshots, if possible).

Have you installed multiple languages? Are you trying to delete the only one existing and default language?

**Comment by florinsith** [comment]
it was fixed in the master allready: https://github.com/opencart/opencart/pull/4722/commits/4bbf3d5f42acc6621c2e3e984551d59c4c5aca6e

**Comment by alaplap** [comment]
Hi,

@andrei-bogdan Yes, I have installed the hungarian language beside english. Hungarian was the default and I wanted to delete the english language.

@florinsith Yes, I saw that, but I think that isn't correct. It seems that the banner descriptions for different languages are stored in the banner_image table instead of the old banner_image_description table.

I am not realy familiar with using github, but I tried to do a request or whatever. Here is my solution to fix it: [banner_image reference added to deleteLanguage()](https://github.com/alaplap/opencart/commit/ea0a1d4bdce9fd419964c7cd56565923e48487b5)
